python/ml4ir/model/metrics/metrics_impl.py

```diff
@@ -163,3 +163,15 @@ def __init__(self, name="ACR", rerank=True, **kwargs):
     def _get_matches_hook(self, y_pred_click_ranks):
         """Return click ranks for MRR"""
         return tf.cast(y_pred_click_ranks, tf.float32)
+
+
+class CategoricalAccuracy(metrics.CategoricalAccuracy):
```

Comment:
question about these "non-public" keras classes.  Can you really not import it anyways?  Why do we need to copy/paste their class?

Reply:
I don't understand. What do you mean by copy/paste their class?